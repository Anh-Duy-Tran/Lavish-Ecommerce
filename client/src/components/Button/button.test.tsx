import { render, screen } from "@testing-library/react";
import { Button } from ".";

describe("Button", () => {
  it("should run this test", () => {
    render(<Button />);
    expect(true).toBe(true);
  });
});
