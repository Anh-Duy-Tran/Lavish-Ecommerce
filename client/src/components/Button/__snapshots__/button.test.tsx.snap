// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledButton Component renders correctly 1`] = `
<DocumentFragment>
  <button
    class="MuiButton-root css-1ebfljq"
    type="button"
    variant="outlined"
  >
    Snapshot
  </button>
</DocumentFragment>
`;
